commands.go

```diff
@@ -464,6 +485,38 @@ func split(ids []string, count int) [][]string {
 	return xs
 }
 
+func doMetrics(c *cli.Context) error {
+	optHostID := c.String("host")
+	optService := c.String("service")
+	optMetricName := c.String("name")
+
+	from := c.Int64("from")
+	to := c.Int64("to")
+	if from == 0 && to == 0 {
```

Comment:
It might be useful if doMetrics automatically set `to` to `now` when only `--from` is specified?